src/handleImportMap.ts

```diff
@@ -32,38 +43,67 @@ export async function handleImportMap(
 async function readImportMap({
   orgKey,
   importMapName,
-}: Params): Promise<ImportMap> {
-  let importMapKV: ImportMap | null = null;
+}: Params): Promise<ImportMap | null> {
+  const kvKey = `import-map-${orgKey}-${importMapName}`;
+
   try {
-    importMapKV = (await MAIN_KV.get(`import-map-${orgKey}-${importMapName}`, {
+    return (await MAIN_KV.get(kvKey, {
       type: "json",
     })) as ImportMap | null;
   } catch (err) {
     // Defensive programming so that we don't take down their production applications
     // if KV has invalid JSON in it or is down
+    console.error(`Error reading import map with key ${kvKey}`);
     console.error(err);
+    return null;
   }
+}
+
+function verifyImportMap(input: ImportMap | null): string[] {
+  const errors: string[] = [];
 
-  let importMap: ImportMap;
+  if (isPlainObject(input)) {
+    const importMap = input as ImportMap;
 
-  if (importMapKV) {
-    importMap = importMapKV;
+    if (isPlainObject(importMap.imports)) {
+      errors.push(...verifyModuleMap(importMap.imports, "importMap.imports"));
```

Comment:
It took me a minute to realize that verifyModuleMap returns an array of errors. So if the array is empty, this conditional doesn't add errors.

Reply:
yep that's what's happening